enum_test.go

```diff
@@ -0,0 +1,52 @@
+package graphql_test
+
+import (
+	"fmt"
+	"reflect"
+	"strconv"
+	"testing"
+)
+
+type benchStringEnum string
+
+func BenchmarkEnumStringStringer(b *testing.B) {
+	v := reflect.ValueOf(benchStringEnum("TEST"))
+	for n := 0; n < b.N; n++ {
+		if s, ok := v.Interface().(fmt.Stringer); ok {
+			s.String()
+		} else {
+			v.String()
+		}
+	}
+}
+
+func BenchmarkEnumStringFmt(b *testing.B) {
+	v := reflect.ValueOf(benchStringEnum("TEST"))
+	for n := 0; n < b.N; n++ {
+		fmt.Sprintf("%s", v)
+	}
+}
+
+type benchIntEnum int
+
+func (i benchIntEnum) String() string {
+	return strconv.Itoa(int(i))
+}
+
+func BenchmarkEnumIntStringer(b *testing.B) {
+	v := reflect.ValueOf(benchIntEnum(1))
+	for n := 0; n < b.N; n++ {
+		if s, ok := v.Interface().(fmt.Stringer); ok {
+			s.String()
+		} else {
+			v.String()
+		}
```

Comment:
Why do we need the `else` statement here? I think that it will never get executed, will it?

Reply:
You're right, I didn't spend much time on this benchmark. Considering it's only used to justify a 1 line change to this PR, should I just remove the benchmark entirely?